src/olympia/devhub/views.py

```diff
@@ -1373,10 +1357,16 @@ def version_list(request, addon_id, addon):
     new_file_form = forms.NewVersionForm(None, addon=addon, request=request)
     is_admin = acl.action_allowed(request, 'ReviewerAdminTools', 'View')
 
+    token = request.COOKIES.get('jwt_api_auth_token', None)
+    if not token:
```

Comment:
You should not need to do this. Add the cookie manually in tests, it should be present in all requests.